odoo_sh/getting_started/branches.rst

```diff
@@ -94,7 +94,7 @@ When you push a new commit in one of these branches,
 a new server is started, with a database created from scratch and the new revision of the branch.
 The demo data is loaded, and the unit tests are performed by default.
 This verifies your changes do not break any of the features tested by them. If you wish, you can
-disable the tests in the :ref:`branch's settings <odoosh-gettingstarted-branches-tabs-mails>`.
+disable the tests allow specific test to b erun with custom tags in the :ref:`branch's settings <odoosh-gettingstarted-branches-tabs-settings>`.
```

Comment:
```suggestion
disable the tests allow specific test to b erun with custom tags in the :ref:`branch's settings
<odoosh-gettingstarted-branches-tabs-settings>`.
```
You can break the line inside a `ref` directive anywhere before the `<`. Preferably after the label.